package.json

```diff
@@ -39,27 +39,28 @@
     "@edx/frontend-component-footer": "10.1.5",
     "@edx/frontend-component-header": "2.3.0",
     "@edx/frontend-platform": "1.11.1",
-    "@edx/paragon": "13.17.5",
+    "@edx/paragon": "16.3.2",
```

Comment:
Should we upgrade `@edx/paragon` to latest (16.5.0)?

Reply:
Oh, sure.  This _was_ latest when I made the PR. ;)

Reply:
How about `16.x`?